rtf

```diff
@@ -416,6 +423,16 @@ class Vagrant(Host):
       m = re.match(r'configure\((?:.*?,){2}\s*' + "'" + self.platform + "'" + r'\s*,\s*(\d+)', line)
       if m:
         port = str(int(m.group(1)) * 100 + 8081)
+        return m.group(1) + "/rudder"
+      #Test if it is an AWS instance
+      else:
+        aws = re.match(r"configure_aws\(.+" + self.platform + "', [0-9]+, '(\S+)'.+", line)
```

Comment:
Why is this regex different from the one 5 lines above ?

Reply:
The first one is trying to define a port number for virtualbox port forwarding, based on the box id and machine id in the current VagrantFile.
The second one is for aws and try to read the machine role to find its public adress. It is mandatory to use the API token generation script. It is also used to execute any rudder-cli commands, which is the purpose of rtf.

Reply:
I means the 2 regex should match the same thing, so why are they so different ?